src/scripts/dev_tools/run_clang_tidy.py

```diff
@@ -174,7 +204,7 @@ def file_matches(file, args):
             return True
     return False
 
-def main(args = None):
+def main(args = None): # pylint: disable=too-many-return-statements
```

Comment:
TBH I think we should just disable that pylint, as already done with the other `too-many-...` warnings

Reply:
I disabled this check on master